### PR TITLE
#36 admin page

### LINKED ISSUE #36: (4) Create design for admin page - extends #5
Given Barry wants to update the inventory
When they navigate to the admin page
Then the page will load matching the design in the figma

**Dev notes**

-Tailwind tables
- Tailwind Trunkate

### PR DESCRIPTION
**Pull request by jonmanahan**
This resolves #36 

Add Admin Page at the static url `localhost:3000/admin`
Update button is present but functionality will be added later.

### CODE REVIEW

File: gilded-rose-llc-frontend/pages/index.tsx [hunk 1]
```diff
@@ -18,26 +17,16 @@ interface HomeProps {
...
+      <Title>available items</Title>
+      <div>
+        <section className="grid grid-cols-3 gap-x-6 gap-y-8">
+          {items.map((item, index) => (
+            <ItemBlock key={index} name={item.name} price={item.price} />
+          ))}
+        </section>
       </div>
-    </div>
+    </>
```
Comment (matteeyao) [inline gilded-rose-llc-frontend/pages/index.tsx:29]: Nice work on the refactoring to components effort!

**Review by matteeyao — APPROVED** [review]
LGTM 🎉

**Comment by mrfishball** [comment]
What does the admin page look like?

**Comment by jonmanahan** [comment]
> What does the admin page look like?

Here is a screenshot of what it looks like, just like the Figma 😄 

image

**Review by mrfishball — APPROVED** [review]

**Pull request merged**